profiles/acrossx.jsonld

```diff
@@ -13,8 +13,9 @@
   },
   "versions": [
     {
-      "id": "https://w3id.org/xapi/acrossx",
-      "generatedAtTime": "2017-07-14T11:35:00Z"
+      "id": "https://w3id.org/xapi/acrossx/v1",
+      "generatedAtTime": "2017-07-14T11:35:00Z",
+      "wasRevisionOf": ["https://w3id.org/xapi/acrossx"]
```

Comment:
the values in wasRevisionOf can only be profile version URIs, not the URI of the overall profile.